examples/complete/secrets.sops.tf

```diff
@@ -0,0 +1,25 @@
+module "secrets" {
```

Comment:
@gberenice I get why you did this, but I think the secrets mixin should only be used within root module and not child modules. We shouldn't require a consumer of this module to use SOPS if they use Vault or SSM Parameter Store or whatever else. We should be agnostic to that. We should use it in `examples/complete` 100%, but I think we should remove it from the child module. What are your thoughts?

Reply:
You did this so resolving -- Thank you!